schema.sql

```diff
@@ -0,0 +1,43 @@
+CREATE DATABASE IF NOT EXISTS airbnb; 
+
+USE airbnb; 
+
+CREATE TABLE users (
+ id INT AUTO_INCREMENT PRIMARY KEY,
+ first_name VARCHAR(20), 
+ last_name VARCHAR(20),
+ image_photo_path VARCHAR(200)
+);
+
+CREATE TABLE rooms (
+ id INT AUTO_INCREMENT PRIMARY KEY,
+ name VARCHAR(30),
+ host_id INT,
+ FOREIGN KEY fk_host(host_id) REFERENCES users(id)
+ ON UPDATE CASCADE
```

Comment:
What is "ON UPDATE CASCADE" used for? Just curious, this doesn't necessarily have to be removed or edited.